source/sessions/driver-sessions.rst

```diff
@@ -463,6 +464,9 @@ opening and authenticating a connection.
 A driver MAY omit a session ID in isMaster commands sent solely for the purposes
 of monitoring the state of a deployment.
 
+A driver MAY omit a session ID in ``KILLCURSORS`` commands intended to reap cursors
+that have gone out of scope.
```

Comment:
Can you add the justification for this please, in an appropriate spot? Deferred GC semantics, etc. =)

Reply:
sure, done